core/src/test/scala/org/apache/spark/SortShuffleSuite.scala

```diff
@@ -50,6 +50,7 @@ class SortShuffleSuite extends ShuffleSuite with BeforeAndAfterAll {
   override def afterEach(): Unit = {
     try {
       Utils.deleteRecursively(tempDir)
+      Utils.clearLocalRootDirs()
```

Comment:
Let's add this in `beforeEach`. This might already be initialised in previous tests.